.github/workflows/build.yaml

```diff
@@ -695,6 +697,9 @@ jobs:
       - name: Scan images for vulnerabilities
         run: |
           release_tag=$(make tag)
+          if [[ ${{ matrix.image }} =~ "operator" ]]; then
+            release_tag=$(make -C operator tag)
+          fi
```

Comment:
as another minor nit / fix: I think it'd be best to use `--force` to ensure we force the scan results to be fresh and not stale; just for the sake of it. Shouldn't make much of a difference on the runtime.